app/server.py

```diff
@@ -36,11 +36,24 @@ def store(url, links):
 def index():
     return render_template('index.html', engines_list=scrapers.keys())
 
+'''
+def bad_request(error):
```

Comment:
this function doesn't handle the case of `csv`

Reply:
@vaibhavsingh97 can this be created as a separate issue for beginners. No case handling for csv too was present earlier. Should I open an issue for the same for beginners?

Reply:
@rupav Yes, That would be great and help beginner to solve the issue as well 👍